tools/plist_to_html/plist_to_html/PlistToHtml.py

```diff
@@ -19,6 +19,8 @@
 from string import Template
 from xml.parsers.expat import ExpatError
 
+from codechecker_common import util
```

Comment:
@csordasmarton what do you think about this module dependency? PlistToHtml.py should be an independent tool. Should we copy the `trim_path_prefixes` function to this tool?

Reply:
This may explain the following error in Travis CI:
`Failure: ImportError (No module named codechecker_common) ... ERROR`

Reply:
I fixed that illegal dependency by passing `trim_path_prefixes` using a `trim_path_prefixes_handler` callback (which can call `codechecker_common.util.trim_path_prefixes`)